.github/workflows/ci-lint.yml

```diff
@@ -43,13 +43,31 @@ jobs:
       - name: Install gitlint
         run: |
           python -m pip install gitlint
+      - name: Install towncrier
+        run: |
+          python -m pip install https://github.com/oasisprotocol/towncrier/archive/oasis-master.tar.gz
+      - name: Check for presence of a Change Log fragment (only pull requests)
+        # NOTE: The pull request' base branch needs to be fetched so towncrier
+        # is able to compare the current branch with the base branch.
+        # Source: https://github.com/actions/checkout/#fetch-all-branches.
+        run: |
+          git fetch --no-tags origin +refs/heads/"${BASE_BRANCH}":refs/remotes/origin/"${BASE_BRANCH}"
+          towncrier check --compare-with "origin/${BASE_BRANCH}"
+        env:
+          BASE_BRANCH: ${{ github.base_ref }}
+        if: github.event_name == 'pull_request'
       - name: Install dependencies
```

Comment:
Unify the order of "Install dependencies" step with the [Explorer's `ci-lint.yml` file](https://github.com/oasisprotocol/explorer/blob/master/.github/workflows/ci-lint.yml).